Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -322,9 +330,9 @@ public void onAdStart(@NonNull String placementReferenceId) {
                 mIsPlaying = true;
                 //Let's load it again to mimic auto-cache
                 if (AdSize.isBannerAdSize(mAdConfig.getAdSize())) {
-                    sVungleRouter.loadBannerAd(mPlacementId, mAdConfig.getAdSize(), mVungleRouterListener);
+                    sVungleRouter.loadBannerAd(mPlacementId, null, mAdConfig.getAdSize(), mVungleRouterListener);
```

Comment:
@HuFangshuai why do we pass mVungleRouterListener here?

Reply:
Seems we passed it before, and avoid the impact for the displaying ad by mIsPlaying value.